src/allotropy/parsers/perkin_elmer_envision/perkin_elmer_envision_parser.py

```diff
@@ -91,11 +91,7 @@ def to_allotrope(self, named_file_contents: NamedFileContents) -> Model:
         lines = read_to_lines(named_file_contents)
         reader = CsvReader(lines)
         filename = named_file_contents.original_file_name
-        try:
-            return self._get_model(Data.create(reader), filename)
-        except Exception as error:
```

Comment:
do we not need to catch this anymore?

Reply:
Sorry should have left a reviewer comment.

This was a bad pattern. They were catching all errors and returning a generic error instead.

The pattern we want is to catch errors where they happen and return an `AllotropeConversionError` with a _helpful_ error message.

Removing this may let some uncaught errors through - but that's how we find and address those uncaught errors.